File: w1/d2/fizzBuzz.js
var fizz = parseInt(process.argv[2])
var buzz = parseInt(process.argv[3])

function print(i){
    console.log(i)
} 

for(var i = 1; i < 21; i ++)
{
    if(i % fizz == 0) print("fizz")
    else if(i % buzz == 0) print("buzz")
    else print(i)
}

File: w5/d2/fizzbuzz.rb
result = []
for x in 1..100
    result.push(x)
end
for x in result
   p "fizz" if x % 3 ==0 
   p "buzz" if x % 5 == 0
   p "fizzbuzz" if x % 3 && x % 5
   p x.to_s if (x % 3) > 0 && (x % 5) > 0
end